src/components/form.js

```diff
@@ -0,0 +1,16 @@
+const Form = () =>{
+    return(
+        <>
+         <form>
```

Comment:
Do not use form instead of this use <div></div>
no need to add br tag try to give space via css

Reply:
i have deleted this one before last commit ,will make use of space with css here onwards